web/assets/css/app.css

```diff
@@ -310,6 +310,12 @@ table.table-samples td {
     font-size:12px;
 }
 
+.eval-revert-form {
```

Comment:
I don't have a better solution at the moment, but this (and the similar class applied to the revert form for biobank orders) should be cleaned up at some point in the future.  Relying on the absolute pixel size is a pretty brittle solution that might not render correctly across browsers/operating systems or if a user is using browser page zoom.